index.html

```diff
@@ -1,173 +1,352 @@
+            <a class="anchorNav" id="anchorIDRegistration" href="#idH1Registration">Regisztráció</a>
+            <a class="anchorNav" id="anchorIDSendArticle" href="#idH1SendArcticle">Cikk küldése</a>
+        </div>
 
-        <h2>Cimsor 2 - Lorem ipsum hosszu</h2>
-            <p>
-                Lorem ipsum dolor sit amet, consectetuer adipiscing elit
-                Aenean commodo ligula eget dolor
-                Aenean massa
-                Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus
-                Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem
-                Nulla consequat massa quis enim
-                Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu
-                In enim justo, rhoncus ut, imperdiet a, venenatis vitae, justo
-                Nullam dictum felis eu pede mollis pretium
-                Integer tincidunt
-                Cras dapibus
-            </p>      
-            <p>
-                Vivamus elementum semper nisi
-                Aenean vulputate eleifend tellus
-                Aenean leo ligula, porttitor eu, consequat vitae, eleifend ac, enim
-                Aliquam lorem ante, dapibus in, viverra quis, feugiat a, tellus
-                Phasellus viverra nulla ut metus varius laoreet
-                Quisque rutrum
-                Aenean imperdiet
-                Etiam ultricies nisi vel augue
-                Curabitur ullamcorper ultricies nisi
-                Nam eget dui
-                Etiam rhoncus
-                Maecenas tempus, tellus eget condimentum rhoncus, sem quam semper libero, sit amet adipiscing sem neque sed ipsum
-                Nam quam nunc, blandit vel, luctus pulvinar, hendrerit id, lorem
-                Maecenas nec odio et ante tincidunt tempus
-                Donec vitae sapien ut libero venenatis faucibus
-                Nullam quis ante
-                Etiam sit amet orci eget eros faucibus tincidunt
-                Duis leo
-                Sed fringilla mauris sit amet nibh
-                Donec sodales sagittis magna
-                Sed consequat, leo eget bibendum sodales, augue velit cursus nunc,
-            </p>
+        <div style="
+                width: 100vw;
+                height:  600px;
+                background-size: cover;
+                background-position: center center;
+                background-repeat: no-repeat;
+                background-image: url(Backgrounds/beautiful-aerial-shot-fronalpstock-mountains-switzerland-beautiful-pink-blue-sky.jpg);
+                vertical-align: middle;
+            ">  
+            <div style="text-align: center;
+                        width: 400px;
+                        margin: auto;
+                        padding-top: 40px;"> <!-- Ez a style kell, h közepre kerüljön a szöveg...-->              
+                <div>
+                    <h1 class="cimStyle">Frontendes hegyes cím  </h1>   
```

Comment:
Vegyesen használsz magyar és angol szavakból álló összetételt és a többi classnévvel sincs szinkronban.
A `Style` a classnévben szóismétlés: a classneveket jellemzően arra használjuk, hogy stílust aggassunk az elemekre.

```suggestion
                    <h1 class="cimStyle">Frontendes hegyes cím</h1>   
```

Reply:
Modositva (commit https://github.com/balog-martin/FrontEndKurzus2023_Hf/commit/9e081b9016871976d4c4f039cb1246675a2b39b1).